js/modules.js

```diff
@@ -541,7 +541,7 @@ new Module("Functions")
 .addAction("endfunction","Ends a function block")
 .addAction("[value =] call(<functionname>,[...parameters])","Calls the function")
 .addAction("[value =] <functionname>([...parameters])","Alias to [value =] call(<functionname>,[...parameters])")
-.addAction("return(<value>,[value 2,value 3...])","Sets the return value of the function and stops it. Use more than one parameter or return an array variable to return more than one information")
+.addAction("return(<value>)","Sets the return value of the function and stops it. Return value can be a constant, a variable, a comma seperated list or an array")
```

Comment:
Keep the optional parameters and specify the "expanded" that I added afterward to prevent confusion